aws-auth-cognito/src/androidTest/java/com/amplifyframework/auth/cognito/testutils/AbortableCountdownLatch.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.auth.cognito.testutils
+
+import java.util.concurrent.CountDownLatch
+
+class AbortableCountdownLatch(count: Int) : CountDownLatch(count) {
```

Comment:
I don't understand the purpose of this class. Why are we not using a normal CountDownLatch?